lib/fastlane/plugin/sentry/actions/sentry_create_deploy.rb

```diff
@@ -0,0 +1,103 @@
+        UI.success("Successfully created deploy: #{version}")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Creates a new release deployment for a project on Sentry"
+      end
+
+      def self.details
+        [
+          "This action allows you to accociate deploys to releases for a project on Sentry.",
+          "See https://docs.sentry.io/product/cli/releases/#creating-deploys for more information."
+        ].join(" ")
+      end
+
+      def self.available_options
+        Helper::SentryConfig.common_api_config_items + [
+          FastlaneCore::ConfigItem.new(key: :version,
+                                       description: "Release version to accociate the deloy with on Sentry"),
+          FastlaneCore::ConfigItem.new(key: :env,
+                                       short_option: "-e",
+                                       description: "Set the environment for this release. This argument is required. Values that make sense here would be 'production' or 'staging'",
+                                       optional: false),
+          FastlaneCore::ConfigItem.new(key: :name,
+                                       short_option: "-n",
+                                       description: "Optional human readable name for this deployment",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :deploy_url,
+                                       description: "Optional URL that points to the deployment",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :started,
+                                       description: "Optional unix timestamp when the deployment started",
+                                       is_string: false,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :finished,
+                                       description: "Optional unix timestamp when the deployment finished",
+                                       is_string: false,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :time,
+                                       short_option: "-t",
+                                       description: "Optional deployment duration in seconds. This can be specified alternatively to `started` and `finished`",
+                                       is_string: false,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :app_identifier,
+                                       short_option: "-a",
+                                       env_name: "SENTRY_APP_IDENTIFIER",
+                                       description: "App Bundle Identifier, prepended to version",
```

Comment:
Can we have line breaks? I wonder if we can make it clear what this is by adding an example:

```suggestion
                                       description: "App Bundle Identifier, prepended with the version.\nFor example bundle@version",
```